packages/test/service-load-test/src/nodeStressTest.ts

```diff
@@ -32,13 +33,14 @@ const codeDetails: IFluidCodeDetails = {
 
 const codeLoader = new LocalCodeLoader([[codeDetails, fluidExport]]);
 
-function createLoader(testDriver: ITestDriver) {
+function createLoader(testDriver: ITestDriver, runId?: number) {
     // Construct the loader
     const loader = new Loader({
         urlResolver: testDriver.createUrlResolver(),
         documentServiceFactory: testDriver.createDocumentServiceFactory(),
         codeLoader,
-        logger,
+        logger: ChildLogger.create(logger, undefined, {all: {buildId: process.env.FLUID_PIPELINE_BUILD_ID,
```

Comment:
you don't need to add a new variable. see https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#system-variables